homeassistant/components/whirlpool/climate.py

```diff
@@ -112,6 +112,17 @@ def __init__(self, hass, said, name, backend_selector: BackendSelector, auth: Au
         self.entity_id = generate_entity_id(ENTITY_ID_FORMAT, said, hass=hass)
         self._attr_name = name if name is not None else said
         self._attr_unique_id = said
+        self._said = said
+
+    @property
+    def device_info(self) -> DeviceInfo | None:
+        """Device information for Whirlpool 6th Sense air conditioners."""
+        return DeviceInfo(
```

Comment:
This can be set up in the constructor directly?

Reply:
Yes it can (and now is)